app/src/screenshotTest/kotlin/io/homeassistant/companion/android/settings/wear/views/SettingsWearOnboardingViewPreviewsTest.kt

```diff
@@ -0,0 +1,45 @@
+package io.homeassistant.companion.android.settings.wear.views
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.tooling.preview.Preview
+import com.android.tools.screenshot.PreviewTest
+import io.homeassistant.companion.android.common.R as commonR
+
+class SettingsWearOnboardingViewPreviewsTest {
+
+    @PreviewTest
+    @Preview
+    @Composable
+    private fun `Onboarding View checking for devices`() {
+        SettingsWearOnboardingViewContent (
```

Comment:
Wrap the screen into the theme otherwise the color are wrong (you can see it in the screenshots)